scripts/upload_ci_stats.sh

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env bash
+set -xue
+
+PSQL=(psql "${CI_STATS_DB:-}")
+
+if ! [ -e 'installcheck.log' ]
+then
+    # Probably the previous steps have failed and we have nothing to upload.
+    echo "installcheck.log does not exist"
+    exit 0
+fi
+
+# The tables we are going to use.
+DESIRED_SCHEMA="
+create extension if not exists timescaledb;
+
+create table if not exists job(
+    job_date timestamptz, -- Serves as a unique id.
+    commit_sha text,
+    job_name text,
+    repository text,
+    ref_name text,
+    event_name text,
+    pr_number int,
+    job_status text,
+    url text,
+    run_attempt int,
+    run_id bigint,
+    run_number int
+);
+
+create unique index on job(job_date);
+
+select create_hypertable('job', 'job_date');
+
+create table if not exists test(
+    job_date timestamptz,
+    test_name text,
+    test_status text,
+    test_duration float
+);
+
+create unique index on test(job_date, test_name);
```

Comment:
I think it's convenient to have unique names for our tests. We had some duplicates (e.g. an isolation and a regression test with the same name), so I renamed them.